libkbfs/tlf_handle.go

```diff
@@ -235,7 +248,7 @@ func (h TlfHandle) Extensions() (extensions []tlf.HandleExtension) {
 }
 
 func init() {
-	if reflect.ValueOf(TlfHandle{}).NumField() != 8 {
+	if reflect.ValueOf(TlfHandle{}).NumField() != 9 {
```

Comment:
Hmm, what is this check doing? I.e. isn't this tautalogical once we've compiled?

Reply:
See the text of the panic just below this -- the check was responsible for making me remember I had to update the `Equals` function to check the tlf ID.